src/components/pages/Notes/Note.js

```diff
@@ -1,34 +1,29 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { Card, Form } from 'react-bootstrap';
 import '../../../styles/notes.css';
-import NoteDialog from './NoteDialog';
+import useNotes from '../../../contexts/NotesContext';
 
-const Note = props => {
-  const [dialogVisible, setDialogVisible] = useState(false);
-  const toggleDialog = e => setDialogVisible(!dialogVisible);
-
-  const { content } = props;
+const Note = ({ note }) => {
+  const [, , { openDialog }] = useNotes();
 
   // TODO - note title and timestamp props
-  const title = 'Note Tilte';
-  const timestamp = Date.now();
   return (
     <>
       <Card
         style={{ backgroundColor: '#f8f9fa', cursor: 'pointer' }}
```

Comment:
This should be in `notes.css`.

Reply:
I've made the exact same comment in Kuba's pr